component/client/core/ClientCore.tsx

```diff
@@ -101,8 +101,8 @@ class ClientCore extends React.Component<any, ClientCoreState> {
 
   private handleSave(): void {
     storage.set(STORAGE_KEY_USERNAME, this.state.username)
-    .then(storage.set(STORAGE_KEY_INITIALIZED, true)
-      .then(this.setState({initialized: true, username: ""})));
+      .then(() => storage.set(STORAGE_KEY_INITIALIZED, "true")
```

Comment:
Check here
https://github.com/zhoutwo/url_connect/blob/9270d989d8ee9fede2f3fb3ef8ffad6f52bfde70/component/background/StorageService.tsx#L34
Since get return a promise of string, 
https://github.com/zhoutwo/url_connect/blob/9270d989d8ee9fede2f3fb3ef8ffad6f52bfde70/component/background/StorageService.tsx#L48
set takes an string.